src/main/java/org/broadinstitute/hellbender/tools/copynumber/DetermineGermlineContigPloidy.java

```diff
@@ -28,73 +28,113 @@
+ *     be used later on for karyotyping one or more similarly-sequenced samples (see below).
  *
- *      The calls subdirectory contains one subdirectory for each sample, listing various sample-specific
- *      quantities such as the global read-depth, average ploidy, per-contig baseline ploidies, and per-contig
- *      coverage variance estimates.</p>
- *     </dd>
+ *     The calls subdirectory contains one subdirectory for each sample, listing various sample-specific
+ *     quantities such as the global read-depth, average ploidy, per-contig baseline ploidies, and per-contig
+ *     coverage variance estimates.</p></dd>
  *
  *     <dt>CASE mode:</dt>
- *     <dd>If a directory containing previously inferred ploidy-model parameters is provided via the {@code model}
- *      argument, then the tool is run in the CASE mode. In this mode, the ploidy-model parameters are loaded from the
- *      provided directory and only sample-specific quantities are inferred. Subsequently, the output directory will
- *      only contain the "-calls" subdirectory.
+ *     <dd>If a directory containing previously inferred ploidy model parameters is provided via the
+ *     <code>--model &lt;path&gt;</code> argument, then the tool will run in the CASE mode. In this mode, the
+ *     parameters of the ploidy model are loaded from the provided directory and only sample-specific quantities are
+ *     inferred. Subsequently, the output directory will only contain the "-calls" subdirectory.
  *
  *      <p>In the CASE mode, the contig ploidy prior table is taken directly from the provided model parameters
  *      path and must be not provided again.</p></dd>
  * </dl>
  *
+ * <h3>Important Remarks</h3>
+ * <dl>
+ *     <dt>Choice of hyperparameters:</dt>
+ *     <dd><p>The quality of ploidy model parametrization and the sensitivity/precision of germline karyotyping are
+ *     sensitive to the choice of model hyperparameters, including standard deviation of mean contig coverage bias
+ *     (set by <code>--mean-bias-standard-deviation &lt;value&gt;</code>), mapping error rate
+ *     (set by <code>--mapping-error-rate &lt;value&gt;</code>), and the typical scale of contig- and sample-specific
+ *     unexplained variance (set by <code>--global-psi-scale &lt;value&gt;</code> and <code>--sample-psi-scale
+ *     &lt;value&gt;</code>, respectively). It is crucial to note that these hyperparameters are <em>not</em> universal
+ *     and must be tuned for each sequencing protocol and properly set at runtime.</p></dd>
+ *
+ *     <dt>Mosaicism and fractional ploidies:</dt>
+ *     <dd><p>The model underlying this tool assumes integer ploidy states (in contrast to fractional/variable ploidy
+ *     states). Therefore, it is to be used strictly on germline samples and for the purpose of sex determination,
+ *     autosomal aneuploidy detection, or as a part of GATK germline CNV calling pipeline. The presence of large somatic
```

Comment:
"of GATK germline" -> "of the GATK germline"

Reply:
Done.